README.md

```diff
@@ -3,3 +3,8 @@
 - all-in-one file so it's easy to download and use offline.
 - mobile friendly form with field validation.
 - somewhat sane fallbacks for users with disabled javascript.
+
+## Generating the application form
+
+With the dependencies in `Pipfile` or `requirements.txt` installed,
+run `./form.py foerder.j2` to generate the `foerder.html`.
```

Comment:
I gave this a quick check and couldn't run this. I didn't really dig into why yet:

```
space@FeTAp /tmp/membership-form (git)-[add-jinja] % ./form.py foerder.j2
Traceback (most recent call last):
  File "./form.py", line 31, in <module>
    form()
  File "/Users/space/.local/share/virtualenvs/membership-form-fqe2t_iN/lib/python3.5/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/Users/space/.local/share/virtualenvs/membership-form-fqe2t_iN/lib/python3.5/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Users/space/.local/share/virtualenvs/membership-form-fqe2t_iN/lib/python3.5/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/space/.local/share/virtualenvs/membership-form-fqe2t_iN/lib/python3.5/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "./form.py", line 26, in form
    template = env.get_template(template_path.name)
  File "/Users/space/.local/share/virtualenvs/membership-form-fqe2t_iN/lib/python3.5/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/space/.local/share/virtualenvs/membership-form-fqe2t_iN/lib/python3.5/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/space/.local/share/virtualenvs/membership-form-fqe2t_iN/lib/python3.5/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/Users/space/.local/share/virtualenvs/membership-form-fqe2t_iN/lib/python3.5/site-packages/jinja2/loaders.py", line 170, in get_source
    filename = path.join(searchpath, *pieces)
  File "/Users/space/.local/share/virtualenvs/membership-form-fqe2t_iN/lib/python3.5/posixpath.py", line 89, in join
    genericpath._check_arg_types('join', a, *p)
  File "/Users/space/.local/share/virtualenvs/membership-form-fqe2t_iN/lib/python3.5/genericpath.py", line 143, in _check_arg_types
    (funcname, s.__class__.__name__)) from None
TypeError: join() argument must be str or bytes, not 'PosixPath'
```

Reply:
Sounds like the template loader can't handle `Path` objects in Python 3.5. I'll pass a string, let me know how that works.

Reply:
Ah hm, don't bother. Depending on 3.6 should be fine, my fault for ignoring what was clearly stated :S. Maybe good to mention it in the README though.

Reply:
According to Jinja documentation it's what the constructor expects, so I'm not actually sure why it was working for me...